ConfigSpace/hyperparameters/numerical.pyx

```diff
@@ -0,0 +1,85 @@
+from typing import Any, Dict, Optional, Union
+
+import numpy as np
+cimport numpy as np
+np.import_array()
+
+
+cdef class NumericalHyperparameter(Hyperparameter):
```

Comment:
This file seems to be a bit less consistent, where the `cimport Hyperparameter` is in the corresponding `.pyd`. I see that it's needed in the `.pyd` but perhaps we can also import it here so it doesn't look as magical?

Reply:
I could add this, but I dropped this on purpose because it is not necessary.

Reply:
No problem, leave it out!